stage1/init/pod.go

```diff
@@ -168,10 +160,10 @@ func (p *Pod) appToSystemd(ra *schema.RuntimeApp, interactive bool) error {
 	}
 
 	env := app.Environment
-	env.Set("AC_APP_NAME", name)
+	env.Set("AC_APP_NAME", appName.String())
```

Comment:
@jonboulle I plan to do this in following up PRs. Another place I think we need to change is the on disk store, where we use appname to reference image name. Can you think of any other places?